boards/x86/qemu_x86/qemu_x86_iamcu_defconfig

```diff
@@ -20,3 +20,4 @@ CONFIG_X86_IAMCU=y
 CONFIG_XIP=y
 CONFIG_X86_MMU=y
 CONFIG_HW_STACK_PROTECTION=y
+CONFIG_X86_PAE_MODE=y
```

Comment:
I'd like to think we are testing both regular ia32 and PAE page tables.
Suggest leaving this option disabled for IAMCU, so we cover both.